### ISSUE TITLE
Requirements: upgrade `django-structlog` to its latest version

### ISSUE THREAD

**Issue by humitos** [readthedocs/readthedocs.org#10933]
We are currently using `2.2.0` but the latest release is `6.x`. We pause upgrading it because it required some extra work by that time. Maybe, due to #10932 the time for updating it has arrived.

**Comment by humitos** [comment]
Maybe related to:

* https://github.com/readthedocs/readthedocs.org/pull/11003

**Comment by agjohnson** [comment]
I think you're working on this right now @humitos so bumped up on the roadmap. Close this if you want though.

**Comment by humitos** [comment]
Yeah, I did a quick try upgrading this to check if the TemplateSyntaxError was faster, but it was the same.

So, there is no rush on moving forward with this now. We can wait if we want.

**Comment by ericholscher** [comment]
This will likely be required when we upgrade Python or Django.
